jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/KindFilenameMapperUtilTest.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.kit.common.util;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayInputStream;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class KindFilenameMapperUtilTest {
+    private static final String MAPPING_PROPERTIES = "Pod=pd, pod";
+
+    private static final String VALID_TABLE = "cols=2*,options=\"header\"]\n"
+            + "|===\n"
+            + "|Kind\n"
+            + "|Filename Type\n\n"
+            + "|BuildConfig\n"
+            + "a|`bc`, `buildconfig`\n\n"
+            + "|ClusterRole\n"
+            + "a|`cr`, `crole`, `clusterrole`\n\n"
+            + "|ConfigMap\n"
+            + "a|`cm`, `configmap`\n\n"
+            + "|CronJob\n"
+            + "a|`cj`, `cronjob`\n"
+            + "|===";
+
+    @Test
+    public void shouldLoadMappings() {
+        // given
+        final AsciiDocParser asciiDocParser = new AsciiDocParser();
+        final Map<String, List<String>> serializedMappings = asciiDocParser.serializeKindFilenameTable(new ByteArrayInputStream(VALID_TABLE.getBytes()));
```

Comment:
Finally this is just checking the location, not the content, as it is using the same code to parse the files.
What if there is a bug in asciiDocParser.serializeKindFilenameTable not parsing correctly? we won't be able to reveal the problem with this test right?

Reply:
I'm really very thankful for you helping me, 
I will look into it